topics/contributing/tutorials/github-contribution/tutorial.md

```diff
@@ -121,6 +168,32 @@ When you're working on a project, you're going to have a bunch of different feat
 
 Here for this tutorial, you will create a branch called "my_new_branch" in which you will modify the `CONTRIBUTORS.yaml` file, the file used to generate the [Hall of Fame]({% link hall-of-fame.md %}).
 
+<div class="Desktop" markdown="1">
```

Comment:
When I [preview the changes](https://github.com/galaxyproject/training-material/blob/d48c0577a73157add00b820e59cd1960a336ca46/topics/contributing/tutorials/github-contribution/tutorial.md#create-a-new-branch), this section does not display correctly on the GitHub website. I assume it worked when you were testing it. 
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/82df6535-33b1-4273-9df0-66cbf3a6e4d5" />

Reply:
Will check! I've added a line that I hope will fix this...

Reply:
That seems to have worked for me and fixed the preview display. Perhaps try the same fix in the second place this happens too?